src/layout/NestedPicker.js

```diff
@@ -28,20 +28,29 @@ let FilteredSection = observer(
   )
 )
 FilteredSection.displayName = 'FilteredSection'
-let Section = observer(({ options, onClick, selected, Item }) => (
+let Section = observer(({ options, onClick, selected, Item = 'div' }) => (
```

Comment:
You should drop this and just check if `Item` is truthy then `Item` else `div`.

Reply:
I can't. React treats uppercase props specially and it complains if `Item` is ever undefined:
```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports. Check the render method of `Section`.
```

Reply:
Just go with your previous solution.